.github/CODEOWNERS

```diff
@@ -11,6 +11,7 @@ x-pack/test/alerting_api_integration/common/plugins/actions_simulators @elastic/
 packages/kbn-actions-types @elastic/response-ops
 src/plugins/advanced_settings @elastic/appex-sharedux @elastic/kibana-management
 x-pack/packages/kbn-ai-assistant @elastic/search-kibana
```

Comment:
should these new packages also be owned by the obs-ai-assistant team?

Reply:
I'd like to keep this with search-kibana for now so we can make progress and not require obs assistant reviews for every separate PR. In the long term these will need to be owned by AppEx as shared components in one way or another